source/MaterialXTest/ShaderValid.cpp

```diff
@@ -862,6 +868,10 @@ bool getTestOptions(const std::string& optionFile, ShaderValidTestOptions& optio
                     {
                         options.shaderInterfaces = val->asA<int>();
                     }
+                    else if (name == "validateElementToRender")
```

Comment:
I notice I never complained about it before, but should these strings be constants?